### PR TITLE
Add a step to install cluster local gateway when setting up cluster

### PR DESCRIPTION
**Pull request by ZhiminXiang**
Fixes https://github.com/knative/docs/issues/1842

## Proposed Changes
- Add a step to install cluster local gateway when setting up cluster

### CODE REVIEW

File: docs/install/installing-istio.md [hunk 1]
```diff
@@ -247,6 +247,22 @@ helm template --namespace=istio-system \
...
+# Istio minor version should be 1.2 or 1.3
+export ISTIO_MINOR_VERSION=1.2
+
+export VERSION=$(curl https://raw.githubusercontent.com/knative/serving/master/third_party/istio-${ISTIO_MINOR_VERSION}-latest)
+
+kubectl apply -f https://raw.githubusercontent.com/knative/serving/master/third_party/${VERSION}/istio-knative-extras.yaml
+```
+
+**Note:** Above installation method is only for development purpose. The production readiness of above 
+installation method is not ensured. For production installtion, refer the `helm` installtion method in this
```
Comment (vaikas-google) [inline docs/install/installing-istio.md]: nit: typo: installtion => installation (times two)

Comment (ZhiminXiang) [reply docs/install/installing-istio.md]: Done

**Comment by vaikas-google** [comment]
@RichieEscarez cool with this going in?

**Pull request merged**